.ddev/docker-compose.testing.yml

```diff
@@ -0,0 +1,26 @@
+version: '3.6'
+services:
+  web:
+    links:
+      - chromedriver:$DDEV_HOSTNAME
+    environment:
+      SYMFONY_DEPRECATIONS_HELPER: weak
+      SIMPLETEST_DB_MYSQL: &simpletest_db_mysql mysql://db:db@db:3306/db
+      SIMPLETEST_DB_SQLITE: &simpletest_db_sqlite sqlite://tmp/sites/simpletest/simpletest.db
+      SIMPLETEST_DB: *simpletest_db_sqlite
+      SIMPLETEST_BASE_URL: &base_url http://${DDEV_HOSTNAME}
+      MINK_DRIVER_ARGS_WEBDRIVER: &mink_driver_args '["chrome", {"browserName":"chrome","goog:chromeOptions":{"args":["--disable-gpu","--headless", "--no-sandbox", "--disable-dev-shm-usage"], "w3c": false}}, "http://chromedriver:9515"]'
+      BROWSERTEST_OUTPUT_DIRECTORY: /var/www/html/web/sites/simpletest/browser_output
+      BROWSERTEST_OUTPUT_BASE_URL: ${DDEV_PRIMARY_URL}
+      DTT_BASE_URL: *base_url
+      DTT_MINK_DRIVER_ARGS: *mink_driver_args
+
+  chromedriver:
+    image:  drupalci/webdriver-chromedriver:production
```

Comment:
Please don't use this image. It works on amd64 machines only, doesn't work on mac M1. Please see (and use) https://github.com/drud/ddev-selenium-standalone-chrome (`ddev get drud/ddev-selenium-standalone-chrome`), which is maintained by @weittzman and has automated tests, and with works on arm64.

Reply:
@rfay : Thanks for the review.

I like the idea of it but it seems the image doesn't have support for SQLite DB. I tried to raise a PR [#6](https://github.com/drud/ddev-selenium-standalone-chrome/pull/6) for its support. 

I tried to override `SIMPLETEST_DB` env variable in `config.yml` as below 

```
composer_version: "2"
web_environment:
- SIMPLETEST_DB=sqlite://tmp/sites/simpletest/db.sqlite
nodejs_version: "16"
```

but it has not overrided the value defined in `docker-compose.selenium-chrome.yaml` do you have any suggestion on how to handle it ?

Reply:
As discussed over there, `docker-compose.*.yaml` wins. But you can do it with a docker-compose.override.yaml here, or work a little more over there to use `config.*.yaml`

I'm not sure why you want to force the user to use sqlite in the first place though. Why is that? IMO tests are less valid when using a little-used database type.

Reply:
> I'm not sure why you want to force the user to use sqlite in the first place though. Why is that? IMO tests are less valid when using a little-used database type.

I am not forcing to use SQLite. I am trying to provide an option for the users to override easily .
And also SQLIte DB tests are faster in compared to MySQL DB, sometimes simpletests leave tables orphaned when the tests failed in MySQL. As we are using our main MySQL DB for tests as well, orphaned tables are present in the main DB itself, which doesn't look good.